src/utils/privateKey.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Does anything break if extra info is added to this file? I think we should somehow more clearly indicate that this is a key generated just for this reference implementation, and it's not a leaked secret. If it doesn't break let's add a "_notes" or some kind of field at the top explaining that

Reply:
I added a readme, I got an error after adding a "_notes" files